features/importing/archivist.feature

```diff
@@ -230,13 +230,69 @@ Feature: Archivist bulk imports
     When I choose "Please remove my works from the archive entirely."
       And I check "Do not email me in the future when works are imported with this email address."
       And I press "Update"
-    Then I should see "Your imported stories have been deleted. Your preferences have been saved."
+    Then I should be on the homepage
+      And I should see "Your imported stories have been deleted. Your preferences have been saved."
     When the email queue is clear
       And I am logged in as "archivist"
       And I import the work "http://ao3testing.dreamwidth.org/325.html" by "randomtestname" with email "random@example.com"
       And the system processes jobs
      Then 0 emails should be delivered to "random@example.com"
 
+  Scenario: Leave an imported work in the archivist's care
+    When I import the work "http://ao3testing.dreamwidth.org/593.html" by "randomtestname" with email "random@example.com"
+      And the system processes jobs
```

Comment:
This step doesn't do anything anymore, we shouldn't use it ([AO3-5722](https://otwarchive.atlassian.net/browse/AO3-5722)).